pkg/kudoctl/util/kudo/upgrade.go

```diff
@@ -54,7 +54,7 @@ func UpgradeOperatorVersion(kc *Client, newOv *v1beta1.OperatorVersion, instance
 	}
 
 	if err = kc.UpdateInstance(instanceName, namespace, convert.StringPtr(newOv.Name), parameters, nil); err != nil {
-		return fmt.Errorf("failed to update instance for new OperatorVersion %s", newOv.Name)
+		return fmt.Errorf("failed to update instance for new OperatorVersion %s: %q", newOv.Name, err)
```

Comment:
s/%q/%v/ ? Looks like "%q" provides nice double-quoted string formatting if the input is a string. But here the input is `error`, so this probably won't print what we expect.

Reply:
typo 👍